test/signalrclienttests/websocket_transport_tests.cpp

```diff
@@ -394,19 +362,20 @@ TEST(websocket_transport_disconnect, disconnect_is_no_op_if_transport_not_starte
     ASSERT_FALSE(close_called);
 }
 
-TEST(websocket_transport_disconnect, exceptions_from_outstanding_receive_task_observed_after_websocket_transport_disconnected)
+// This is an odd test
+TEST(websocket_transport_disconnect, DISABLED_exceptions_from_outstanding_receive_task_observed_after_websocket_transport_disconnected)
```

Comment:
So why is this disabled?

Reply:
We might want to delete the test. I think it was checking that a crash didn't happen back when we used `pptx::task`, as that would crash for unobserved exceptions in tasks.

Or we might want to refactor this to check that an exception is logged/bubbled up.